### DISCUSSION TITLE
ASIO4ALL issue with external soundcard (AG06)?

### DISCUSSION THREAD

**Discussion by ngocdh** [jamulussoftware/jamulus#1276] [Technical Support & Help]
**Describe the bug**
I'm not sure if this is a ASIO4ALL bug or it has something to do with the Input Channel Mapping.
I'm on Windows 10 and have a Yamaha AG06 soundcard. I wanted to try different settings with inputs outputs using ASIO4ALL (v2.14).
However, if I choose to use inputs/outputs from the soundcard thru ASIO4ALL, all I get is inaudible noise. Anyone has experience with solving this issue?

**To Reproduce**
- In settings window, choose ASIO4ALL v2 as device, and click "ASIO Device Settings"
- In ASIO4ALL window, enable built-in input/output and soundcard input/output (or just the soundcard)
- Now in settings window, you'll see the Input Channel Mapping interface: choose any inputs and outputs
- Notice the output is just inaudible noise

**Expected behavior**
Possibility to use different inputs and hear clear outputs.

**Screenshots**
![image](https://user-images.githubusercontent.com/6765612/111205762-e280a580-85c7-11eb-9d1d-364b4c8f1978.png)

**Operating system**
Windows 10 latest 

**Version of Jamulus**
3.6.2dev-afd47a83-hoffie-inputgain (for the input gain possibility, but during tests the input gain was set to 100%, the default value)

**Comment by ann0see** [comment]
I think there are multiple issues with ASIO4ALL we'll need to address in a new version. Unfortunatly I'm not sure how much we can do there. It might be ASIO4ALL's fault, but I can't comment on that.

**Comment by npostavs** [comment]
> . I wanted to try different settings with inputs outputs using ASIO4ALL (v2.14).
> However, if I choose to use inputs/outputs from the soundcard thru ASIO4ALL, all I get is inaudible noise

Do you mean you always get noise when using the AG06 with ASIO4ALL, or is just when you try to mix multiple devices together?

For reference, I have a MOTU M2, and it seems to work fine with ASIO4ALL, individually or together with my builtin Realtek soundcard (though the builtin microphone is on the quiet side).

**Comment by ann0see** [comment]
@ngocdh @npostavs 

Just converted this to a discussion.
